clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,153 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 5}}
+{{$MEDIUM_BACKEND_SIZE := 3}}
+{{$SMALL_BACKEND_SIZE := 1}}
+{{$LARGE_BACKEND_LABEL := "ilblarge"}}
+{{$MEDIUM_BACKEND_LABEL := "ilbmedium"}}
+{{$SMALL_BACKEND_LABEL := "ilbsmall"}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
+name: ilbload
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: ILBConstantQPS
+  qpsLoad:
+    averageQps: {{$ilbQPS}}
+steps:
+# Create ILBs
+- measurements:
+  - Identifier: ServiceCreationLatencyLarge
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+      labelSelector: size = {{$LARGE_BACKEND_LABEL}}
+  - Identifier: ServiceCreationLatencyMedium
```

Comment:
Can you add a short comment why we want to measure those separately, and not together?

Reply:
done